build/zip.py

```diff
@@ -32,6 +32,8 @@
   # On Linux, we don't use crashpad, but this binary is still built for some
   # reason. Exclude it from the zip.
   './crashpad_handler',
+
+  'resources/inspector', #Skipping because these are outputs that we don't need
```

Comment:
Added context, these outputs are already bundled into `devtools_resources.pak` hence we don't need them.